test.py

```diff
@@ -1099,6 +1099,37 @@ def test_fetch_malformed_url(self):
 
         self.assertEqual(expected_msg, str(cm.exception))
 
+    def test_bag_symlink_is_dangerous(self):
+        src = j(os.path.dirname(__file__), "README.rst")
+        dst = j(self.tmpdir, "README.rst")
+        os.symlink(src, dst)
+        self.assertRaisesRegex(
```

Comment:
I think it must be assertRaisesRegex**p** to work in 2.7

Reply:
Ah, yes -- nice catch!